### DISCUSSION TITLE
How To obtain the temperature data of the hot bed and nozzle in ESP3D

### DISCUSSION THREAD

**Discussion by liqijian101** [luc-github/ESP3D#922] [F.A.Q] [answered]
At present, I can see the temperature data of the hot bed and nozzle on webui. I want to obtain the temperature data of the hot bed and nozzle in ESP3D and send the data to IOT. May I know which variable or code I can use.

**Comment by luc-github** [comment, marked as answer]
Currently ESP3D does not  see/interprete any data (temperature / positions/ etc) it is just a bridge -  so it is not possible currently

ESP3D-TFT project will be more standalone solution because it need to display information on screen but it is not ready yet
